src/app/parcels/ActionBar/ActionsModal.tsx

```diff
@@ -163,9 +171,17 @@ export const DayOverviewInput: React.FC<DayOverviewInputProps> = ({
     );
 };
 
+const deleteParcels = async (parcels: ParcelsTableRow[]): Promise<void> => {
+    await saveParcelStatus(
+        parcels.map((parcel) => parcel.parcelId),
+        "Request Deleted"
+    );
+    return;
+}; //assume successful for now
+
 const ActionsModal: React.FC<ActionsModalProps> = (props) => {
-    const [loadPdf, setLoadPdf] = useState(false);
     const [loading, setLoading] = useState(false);
+    const [loadPdf, setLoadPdf] = useState(false);
```

Comment:
I'm not sure I fully understand the intention of these two states. It seems like
- we set both to false
- we set loading to true when we click on the action button
- then when loading is true and loadPdf is false (which is guaranteed to happen at this point), then we toggle both of them.

It seems like in essence this captures two states - (1) we haven't started the action (2) we have finished the action - in which case we should have just one boolean state. 

Am I missing something?

Reply:
For record
the 2 states are somehow needed for the "Create PDF" button to change to "Loading..." in the time that it takes to render the download PDF secondary page to the modal. In another ticket will look at combining these pages thus removing need for 2 states